Drop-In/src/main/java/com/braintreepayments/api/SelectPaymentMethodChildFragmentAdapter.java

```diff
@@ -0,0 +1,79 @@
+package com.braintreepayments.api;
+
+import android.os.Bundle;
+
+import androidx.annotation.NonNull;
+import androidx.fragment.app.Fragment;
+import androidx.fragment.app.FragmentManager;
+import androidx.lifecycle.Lifecycle;
+import androidx.viewpager2.adapter.FragmentStateAdapter;
+
+import java.util.List;
+
+public class SelectPaymentMethodChildFragmentAdapter extends FragmentStateAdapter {
```

Comment:
Can this be package private?

Reply:
Also should this class have a unit test of some sort?

Reply:
Def!

Reply:
This class used to be a ListViewAdapter though. The unit test didn't translate very well since they have different APIs. I ended up extracting the data portion into a `SelectPaymentMethodChildFragmentList` and added a unit test.